enhancements/kubernetes-nmstate-integration.md

```diff
@@ -57,15 +57,25 @@ what this project is addressing.
 
 ## Proposal
 
-A new kubernetes-nmstate handler DaemonSet is deployed in the cluster part of the OpenShift installation.
+A new kubernetes-nmstate operator Deployment is deployed to the cluster as part of the the OpenShift
+installation, via CVO.
+
+The operator has a single CRD, called `NMstate`. When a custom resource of that type is created,
+it will then create the CRDs for the kubernetes-nmstate handler, including `NodeNetworkState`,
+`NodeNetworkConfigurationPolicy`, and `NodeNetworkConfigurationEnactment`. Then, the namespace,
+RBAC, and finally the DaemonSet are applied.
+
+A new kubernetes-nmstate handler DaemonSet is deployed in the cluster by the operator.
 This DaemonSet contains nmstate package and interacts with the NetworkManager
-on the host by mounting the related dbus. The project contains two
-Custom Resource Definitions, `NodeNetworkState` and `NodeNetworkConfigurationPolicy`.
-`NodeNetworkState` objects are created per each node in the cluster and can be
-used to report available interfaces and network configuration. These objects
-are created by kubernetes-nmstate and must not be touched by a user.
+on the host by mounting the related dbus. The project contains three
+Custom Resource Definitions, `NodeNetworkState`, `NodeNetworkConfigurationPolicy` and
+`NodeNetworkConfigurationEnactment`. `NodeNetworkState` objects are created per each
```

Comment:
Dup

Reply:
I don't understand... What's duplicated?

Reply:
Is all good, I miss readed it.